<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:media="http://search.yahoo.com/mrss/">
    <channel>
        <title>36氪 - 最新资讯频道</title>
        <link>https://www.36kr.com/information/web_news</link>
        
        <item>
            <id>https://www.36kr.com/p/2515418234949892</id>
            <title>黑客入侵ChatGPT，OpenAI微软全被搞崩，“苏丹匿名者”：是我干的</title>
            <link>https://www.36kr.com/p/2515418234949892</link>
            <guid isPermaLink="false">https://www.36kr.com/p/2515418234949892</guid>
            <pubDate></pubDate>
            <updated>Mon, 13 Nov 2023 01:48:25 GMT</updated>
                
                
            <content:encoded>
                <![CDATA[
                    
                        <div> 关键词: 搞炸, AI公司, ChatGPT, 宕机
<br>
<br>
总结: 世界上最先进的AI公司ChatGPT宕机，可能是由于用户过于热情或黑客攻击所致。尽管官方声称问题已解决，但服务中断仍持续，并且用户仍然遇到访问问题。 </div>
                        <hr>
                    
                    <p>搞炸世界上最先进的AI公司，只需要最朴素的手法？黑客组织「苏丹匿名者」宣布对ChatGPT宕机负责！</p><p>ChatGPT的大规模宕机，竟是由于黑客攻击？&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_893311faa9ce4e45a6d487c341be7ea1@5888275_oswg565587oswg1080oswg1064_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>昨天的大规模宕机，把大家都吓坏了。&nbsp;</p><p>ChatGPT在过去24小时内，一直在经历零星的中断。每个试图登录的用户，都会看到这条消息：「Something went wrong.」&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_9b10d0cba53a49a885ca0293e892e2c1@5888275_oswg75246oswg1080oswg597_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>服务断得非常彻底，一直炸了几十个小时，邮箱里连收十几封邮件，简直是前所未有。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_0d5f3b3725c04e9f984e63c247117c59@5888275_oswg188215oswg1080oswg279_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p class="img-desc">来源：歸藏&nbsp;</p><p>需要调用API的套壳公司们，更是吓出一身冷汗。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_a121f8736eb442edb51890effd6d7630@5888275_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>一片混乱之际，Sam Altman冒出来解释道，因为OpenAI开发者大会效果太炸裂，大家太过热情，疯狂涌入的用户们把ChatGPT搞崩了。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_962ed6997e454e7db774bdbad2c55e22@5888275_oswg159858oswg1080oswg456_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>不过，外媒却挖出了另一种可能——这次大规模宕机事件，很可能是由于受到了DDoS攻击。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_dfdcc5d6d6784112aa93590a90a3945c@5888275_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><h2><strong>01 是用户太热情，还是黑客搞事情？</strong></h2><p>原因就在于，在Sam Altman将宕机问题归咎于用户太热情之后，OpenAI表示，问题已经在11月8日太平洋标准时间下午1点得到了解决。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_b3a674339e40498a9454948f75170d17@5888275_oswg191866oswg1080oswg619_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>结果，ChatGPT和API的周期性中断，并没有停止。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_6d996b8184a34e8e81d345f727c5ec6d@5888275_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>而且也有很多用户表示，即使是在官方宣布服务恢复之后，依然还是会遇到访问的问题。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_d607b3ede4ce446b945b69dbcd89618e@5888275_oswg48589oswg732oswg407_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>对此，OpenAI并没有立即做出回应，但根据官方发布公告可以看出，造成后续「间断性的宕机」的原因，的确与DDoS攻击有关。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_0e31985decea4a7384d73b12ac5f98a9@5888275_oswg216338oswg1080oswg711_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>所以，搞炸世界上最先进的AI公司，只需要最朴素的手法？&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_c0dcd7054b3941f981a08802aa44c665@5888275_oswg73067oswg587oswg499_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>而DDos攻击，本质上就是将黑客攻击行为伪装成用户访问，来搞崩服务器。&nbsp;</p><p>所以，在包括GPTs等新功能都还没有完全上线的前天晚上，为啥会有大量用户「疯狂」涌入ChatGPT，多少有点让人疑惑。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_3d668d3b935c431ab6bd643a179433e6@5888275_oswg83276oswg229oswg220_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>很多网友也忍不住发问，这真不是「黑客请的水军」？&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_3ecdfa8e35ff43689a2676055b53ef38@5888275_oswg121997oswg586oswg621_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>巧合的是，除了OpenAI之外，他的老对手Anthropic也出事了。&nbsp;</p><p>在同一时间，Claude也在出现了访问故障。登录后会发现：「由于意外的容量限制，Claude无法回复您的消息。」&nbsp;</p><h2><strong>02 已被黑客认领</strong></h2><p>根据外媒的报导，他们在潜伏的黑客群里看到，有黑客组织「Anonymous Sudan」宣称对这些针对AI公司的攻击负责。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_cac4cc279ee943e0bb860d21c0d8546c@5888275_oswg11273oswg1080oswg40_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>而且他们还非常挑衅地宣布，想看看OpenAI会不会承认他们的宕机是来源于DDos攻击。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_a8c1eb3b8ff74277b0b7a0d55fe9b197@5888275_oswg265141oswg521oswg837_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>对此，有不少网友表示，OpenAI需要提供更多关于故障的信息，才会让用户们更加放心。&nbsp;</p><p>但Sam Altman只是敷衍地表示，没事没事，不用担心（We’re All Gonna Make It）。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_2d08844b4fc04e17845f80d4e5f15605@5888275_oswg148407oswg596oswg904_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><h2><strong>03 OpenAI树大招用户，也招黑客</strong></h2><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_b0e03203e9294856bc788912b0d9cf26@5888275_oswg34266oswg635oswg298_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>OpenAI，果然是树大招风。&nbsp;</p><p>从11月以来，到这次大规模宕机之前，其实OpenAI的各个功能和API都多少出现了一些异常。&nbsp;</p><p>而巧合的是，黑客组织也确认了自10月份以来一直通过一个名为「天网（Skynet）」的系统提供「压力服务」。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_b1ec24883eaf4b6ba057c8cca81f34d5@5888275_oswg44322oswg526oswg196_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>并于上周增加了对应用层攻击或第7层 (L7) DDoS攻击的支持。&nbsp;</p><p>在第7层DDoS攻击中，黑客以应用程序级别为目标，通过大量请求淹没服务，导致服务因无法处理所有请求而宕机。&nbsp;</p><p class="image-wrapper"><img src="https://img.36krcdn.com/hsossms/20231112/v2_4cf1782a3ea94541965e23f113d81e31@5888275_oswg60460oswg526oswg298_img_000?x-oss-process=image/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1/format,jpg/interlace,1" /></p><p>而且，就这个黑客组织，6月份的时候同样也针对微软旗下的一系列服务比如Outlook，One Drive进行了类似的攻击。&nbsp;</p><p>现在，回过头就盯上了当红炸子鸡OpenAI。&nbsp;</p><p>总结来说，这次黑客攻击，也让这样一个问题凸显出来——基础设施的漏洞，正困扰着互联网上数量最庞大、技术最前沿的技术冲浪者们。&nbsp;</p><p>即使你正在开发着改变世界的人工智能，也很难保证网站的日常运营。&nbsp;</p><p>参考资料：&nbsp;</p><p>https://www.bleepingcomputer.com/news/security/openai-confirms-ddos-attacks-behind-ongoing-chatgpt-outages/&nbsp;</p><p class="editor-note">本文来自微信公众号<a href="https://mp.weixin.qq.com/s/iV8_3Qg5ekPGchjn7DoomA" rel="noopener noreferrer nofollow" target="_blank">“新智元”（ID:AI_era）</a>，作者：新智元，36氪经授权发布。</p>
                ]]>
            </content:encoded>
        </item>
        
    </channel>
</rss>